pkg/asset/machines/worker.go

```diff
@@ -162,6 +170,21 @@ func (w *Worker) Generate(dependencies asset.Parents) error {
 				machineSets = append(machineSets, set)
 			}
 		case nonetypes.Name, vspheretypes.Name:
+			// machines are managed directly by users
+			return nil
```

Comment:
again this change shouldn't exist. the master looks like:
https://github.com/openshift/installer/blob/master/pkg/asset/machines/worker.go#L164